defaults/main.yml

```diff
@@ -39,3 +39,5 @@ osbs_build_json_dir: /usr/share/osbs
 osbs_sources_command: fedpkg sources
 osbs_vendor: Fedora Project
 osbs_nodeselector: ''
+osbs_managed_node_labels:
```

Comment:
Why is this not in `operations/defaults/main.yml` ?

Reply:
right, moving to operations